docs/references/keywords.rst

```diff
@@ -194,7 +194,46 @@ extensions).
 .. _keyword/package_dir:
 
 ``package_dir``
-    A dictionary providing a mapping of package to directory names.
+    A dictionary that maps package names (as the developer wishes they would be
+    imported by the end-users) into directory paths (that actually exist in the
+    project's source tree). This configuration can be used with 2 main purposes:
```

Comment:
```suggestion
    project's source tree). This configuration method has two main purposes:
```
I'm not totally sure about saying 'method', but I think this is clearer.

Reply:
Let's just remove it all togheter:

```suggestion
    project's source tree). This configuration has two main purposes:
```